packages/web/src/store/application/ui/buy-audio/sagas.ts

```diff
@@ -934,10 +934,13 @@ function* doBuyAudio({
       console.error('doBuyAudio: unexpectedly no fee payer override')
       return
     }
-    const { currentUser } = yield* select(getWalletAddresses)
+    const { currentUser, web3User } = yield* select(getWalletAddresses)
     if (!currentUser) {
       throw new Error('Failed to get current user wallet address')
     }
+    if (currentUser !== web3User) {
+      throw new Error('Not allowed to recover while in manager mode')
```

Comment:
We're going to run this every time though, right? So we're just guaranteed to throw an error in manager mode?
Feels like this could just be a console.log and early return instead.